cmd/server/api/router_test.go

```diff
@@ -0,0 +1,69 @@
+package api_test
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/freeverseio/laos-universal-node/cmd/server/api"
+	"github.com/freeverseio/laos-universal-node/cmd/server/api/mock"
+	stateMock "github.com/freeverseio/laos-universal-node/internal/platform/state/mock"
+	"github.com/gorilla/mux"
+	"go.uber.org/mock/gomock"
+)
+
+func TestCORS(t *testing.T) {
+	mockCtrl := gomock.NewController(t)
+	defer mockCtrl.Finish()
+
+	mockRPCHandler := mock.NewMockRPCHandler(mockCtrl)
+	state := stateMock.NewMockService(mockCtrl)
+
+	mockRPCHandler.EXPECT().PostRPCRequestHandler(gomock.Any(), gomock.Any()).Times(1)
+	mockRPCHandler.EXPECT().SetStateService(gomock.Any()).Times(1)
+	router := mux.NewRouter()
+	api.Routes(mockRPCHandler, router, state)
+
+	ts := httptest.NewServer(router)
+	defer ts.Close()
+
+	client := ts.Client()
+
+	tests := []struct {
+		method       string
+		url          string
```

Comment:
No need for this parameter, you can hardcode "/" when creating a request

Reply:
I will leave this, maybe we want to test a url at some point